lib/id3c/cli/command/etl/manifest.py

```diff
@@ -55,6 +56,7 @@ def etl_manifest(*, db: DatabaseSession):
     expected_identifier_sets = {
         "samples": {"samples"},
         "collections": {
+            "collections-clia-compliance",
```

Comment:
We have one record from the incident report manifest that has a barcode from the collections-clia-compliance identifier set. This record was recorded on 5/21/2020. 
It is OK to add collections-clia-compliance to the expected set, or was there some sort of barcode mixup with this one?

Reply:
This should not include `collections-clia-compliance`, it must be a barcode mixup in the incident report.

Reply:
Thanks! I removed it.

Reply:
It looks like this didn't get removed as expected.